src/components/listings.jsx

```diff
@@ -18,41 +32,50 @@ export default class Listings extends Component {
         "http://localhost:8080/listings"
       );
       const listingsBack = lisstingDataResponse.data;
-      const listingsWithCordsPromises = listingsBack.map(async listing => {
+      const listingsWithCordsPromises = listingsBack.map(async (listing) => {
         const response = await Geocode.fromAddress(listing.location);
         const { lat, lng } = response.results[0].geometry.location;
         return {
           ...listing,
           lat,
-          lng
+          lng,
         };
       });
       const listingsWithCords = await Promise.all(listingsWithCordsPromises);
       //only way to update a state
       this.setState({
-        listings: listingsWithCords
+        listings: listingsWithCords,
       });
     })();
   }
 
   deleteListing(id) {
     axios
       .delete(`http://localhost:8080/listings/${id}`)
-      .then(res => {
+      .then((res) => {
         console.log("Listing successfully deleted!");
       })
-      .catch(error => {
+      .catch((error) => {
         console.log(error);
       });
 
     this.setState({
-      listings: this.state.listings.filter(listing => listing.id !== id)
+      listings: this.state.listings.filter((listing) => listing.id !== id),
     });
   }
 
   render() {
+    let filteredListings = this.state.listings.filter((listing) => {
+      return listing.name.indexOf(this.state.search) !== -1;
```

Comment:
Make sure this is not case-sensitive. As of now, when I write down 'room' it doesn't show me all the listings that have the word room because all of them are 'Room'.